coordinator/src/services/translation/response/common.ts

```diff
@@ -221,6 +229,11 @@ export function translateAgentPerson(agentPerson: hl7V3.AgentPerson): Translated
 
       return translatedAgentPerson
     } else {
+      const representedOrganization = agentPerson.representedOrganization
```

Comment:
minor but we could pull out `const representedOrganization = agentPerson.representedOrganization` from lines 198 and 232 to be before the above `if` and then use it on line 197